DeviceTests/DeviceTests.Shared/Geolocation_Tests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xamarin.Essentials;
+using Xunit;
+
+namespace DeviceTests
+{
+    public class Geolocation_Tests
+    {
+        /*
```

Comment:
The test are commented out as all platforms require user approval - maybe not the old Androids.